itis.playground/Pages/HW_part2.xcplaygroundpage/Contents.swift

```diff
@@ -0,0 +1,87 @@
+
+import Foundation
+
+class Fraction {
+    var numerator: Int
+    var denominator: Int
+    
+    init(numerator: Int, denominator: Int) {
+        self.numerator = numerator
+        self.denominator = denominator
+    }
+}
+
+var a = Fraction(numerator: 2, denominator: 4)
+var b = Fraction(numerator: 6, denominator: 9)
+var c = Fraction(numerator: 5, denominator: 5)
+var d = Fraction(numerator: 20, denominator: 15)
+
+
+func simplify(fr: Fraction) -> Void {
+    var numeratorHolder = fr.numerator
+    var denominatorHolder = fr.denominator
+    
+    while numeratorHolder != denominatorHolder {
+        if numeratorHolder > denominatorHolder {
+            numeratorHolder -= denominatorHolder
+        } else {
+            denominatorHolder -= numeratorHolder
+        }
+    }
+
+    fr.numerator /= numeratorHolder
+    fr.denominator /= numeratorHolder
+}
+
+func commonDenominator (fr1: Fraction, fr2: Fraction) -> Void {
+    let denominatorHolder = fr1.denominator
+    fr1.denominator *= fr2.denominator
+    fr1.numerator *= fr2.denominator
+    fr2.denominator *= denominatorHolder
+    fr2.numerator *= denominatorHolder
+}
+
+func sum (fr1: Fraction, fr2: Fraction) -> Fraction {
```

Comment:
```suggestion
func sum(fractionLeft: Fraction, fractionRight: Fraction) -> Fraction {
```
хотя бы так, старйся не сокращать названия

Reply:
поправь везде на понятыне проперти

Reply:
ну и пробелы тоже везде, писать не буду сам найдешь